src/input_common/tas/tas_input.h

```diff
@@ -0,0 +1,145 @@
+// Copyright 2020 yuzu Emulator Project
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <array>
+#include <mutex>
+#include <thread>
+
+#include "common/common_types.h"
+#include "core/frontend/input.h"
+#include "input_common/main.h"
+
+namespace TasInput {
+
+constexpr int PLAYER_NUMBER = 8;
```

Comment:
```suggestion
constexpr size_t PLAYER_NUMBER = 8;
```
This avoids the signed mis-match in previous review comments